service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java

```diff
@@ -34,31 +30,16 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Map<String, Object> data = caseDetails.getData();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if ("SA20C50034".equals(caseData.getFamilyManCaseNumber()) && 1600893083735118L == caseDetails.getId()) {
-            data.put("hearingDetails", removeHearingLinkedToCmo(caseData.getHearingDetails(),
-                caseData.getSealedCMOs().get(0).getId()));
+        if ("SN20C50019".equals(caseData.getFamilyManCaseNumber()) && 1603717767912577L == caseDetails.getId()) {
```

Comment:
Checking both is probably a bit unnecessary moving forward as we can guarantee that CCD ID is unique to case. But happy if you feel safer with both considering we are modifying prod data.

Reply:
Removed condition on FamilyManCaseNumber